silk/utils/process.py

```diff
@@ -65,7 +60,7 @@ def process_cmd_asyc(self):
     def read(self, process):
         while not self.stop_thread.is_set():
             output = process.stdout.readline()
-            #TODO: should add the logic to record the log info here
+            # TODO: should add the logic to record the log info here
```

Comment:
with Issue ID link?

Reply:
Hi Jennie,

I have added the issue link ids as suggested. Please let me know if it looks ok.
Thanks in advance.